index.js

```diff
@@ -1 +1,30 @@
-// TODO: Start prototyping here
\ No newline at end of file
+const express = require('express');
+const app = express();
+
+app.get('/', (req, res) => {
+    res.sendFile(`${__dirname}/index.html`);
```

Comment:
It's better to use constants for any literals.  Tests are the only exception to this rule since literals in tests usually seem more readable than constants. Also, `path.join` is recommended for constructing paths.